lib/find-scaffolds.js

```diff
@@ -34,9 +32,12 @@ const loaded = (resolve, reject) => (error) => {
 
 
 // TODO: add caching
-const findScaffolds = () =>
-  // Configure npm to search for dependencies globally with minimal depth
-  new Promise((...args) => npm.load({global: true, silent: true, depth: 0}, loaded(...args)));
+const findScaffolds = npm => global => () =>
+  // Configure npm to search for dependencies globally or locally with minimal depth
+  new Promise((...args) => npm.load({global, silent: true, depth: 0}, loaded(npm)(...args)));
+
+
+export const findLocal = npm => findScaffolds(npm)(false);
```

Comment:
Kill the [boolean trap](http://ariya.ofilabs.com/2011/08/hall-of-api-shame-boolean-trap.html), please. Switch on `'local'`/`'global'` instead of `true`/`false`.